src/components/shared/ProfileModal.tsx

```diff
@@ -204,22 +234,20 @@ const Shared = forwardRef<Ref, Props>((props, ref) => {
                   justifyContent: 'center',
                 }}
               >
-                <StyledImage
-                  source={IC_NO_IMAGE}
-                />
+                <Ionicons name="ios-person" size={80} color="white" />
               </View>
             )}
           </TouchableOpacity>
-          <StyledTextDisplayName
-            numberOfLines={1}
-          >{nickname}</StyledTextDisplayName>
+          <StyledTextDisplayName numberOfLines={1}>
+            {nickname}
+          </StyledTextDisplayName>
           <StyledTextstatusMessage>{statusMessage}</StyledTextstatusMessage>
         </StyledView>
         {isFriendAdded ? (
           <StyledTextFriendAdded testID="added-message">
             {getString('FRIEND_ADDED')}
           </StyledTextFriendAdded>
-        ) : isFriendAlreadyAdded ? (
+        ) : data?.friends?.find((friend) => friend.id === user?.id) ? (
```

Comment:
I'd recommend to retrieved `friends` before and remove optional chaining here. This is also difficult to test later on not only affect on readability.

Also, I recommend you to use ternary operators in rendering like below.
```
{
  isFriendAdded
  ? ...
  : ...
     ? ...
      : ...
```
Above is more general way of using in in render function which I think is more readable.

Reply:
so optional chaining is not proper to use in this **hackatalk-mobile** project? if so, changing another optional chaining codes in this PR is also needed.

Reply:
Optional chaining is fine! I was only mentioning on the ternary statements :)

Reply:
resolved by [f9e2b0e](https://github.com/dooboolab/hackatalk-mobile/pull/115/commits/f9e2b0e4a6ede2e9fe761d32f81e4e6ca5056bd2)